docs/content/getting-started/using-pfelements.md

```diff
@@ -0,0 +1,179 @@
+
+The list below is the recommended approach to using web components and overriding style properties, in order.
+
+
+## 1. Markup
+
+
+### **1a. Content Components**
+
+The beauty of web components is that they have much of the styling built-into the tag itself. Start with the tags first. **And please use named slots! **
+
+
+```
+
+      <pfe-card>
+        <h2 slot="pfe-card--header">Light card</h2>
+  <pfe-cta>hello</pfe-cta>
+      </pfe-card>
+```
+
+
+
+### **1b. Container components (see 
+
+<p id="gdcalert7" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: undefined internal link (link text: "container"). Did you generate a TOC? </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert8">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
+
+[container](#heading=h.2pji8rj06i0b) notes below)**
+
+
+```
+    <pfe-card pfe-color="darkest">
+      <pfe-cta>hello</pfe-cta>
+    </pfe-card>
+```
+
+
+
+## 2. Attributes
+
+Check out [storybook](https://patternfly.github.io/patternfly-elements/demo) to see what attributes are readily available
+
+
+### **2a. General**
+
+Priority & variant attributes change style styles globally
+
+
+```
+    <pfe-cta pfe-priority="primary">hello</pfe-cta>
+    <pfe-tabs pfe-variant="earth">...</pfe-tabs>
+```
+
+
+
+#### **Full list:**
+
+
+
+*   `pfe-priority`
+*   `pfe-variant`
+
+
+### **2b. Specific**
+
+Color or align attributes change specific properties
+
+
+```
+<pfe-cta pfe-priority="primary" pfe-color="accent">
+```
+
+
+
+#### **Full list:**
+
+
+
+*   `pfe-align`
+*   `pfe-color`
+*   `vertical`
+*   `horizontal`
+
+
+## 3. CSS variables
+
+
+### **3a. Page-level CSS, [theme](https://static.redhat.com/libs/redhat/redhat-theme/2/advanced-theme.css) variables**
+
+Theme variables will impact all components on the page where this CSS is loaded.
+
+
+```
+  your-page.css
+    :root {
+      --pfe-theme--color--ui-accent: green;
+  --pfe-theme--color--surface--darker: navy;
+    }
+```
+
+
+_Note_: overriding local variables (i.e., --pfe-cta--foo) will not work at this level.  Those overrides can only be done on the component via tag name, class, or ID.
+
+
+### **3b. Page-level CSS, component variables**
+
+
+```
+  your-page.css
+    pfe-cta {
+      --pfe-cta--BackgroundColor: green;
+    }
+pfe-band {
+      --pfe-band--Padding--vertical: 34px;
+}
+```
+
+
+
+### **3c. Component-level, inline CSS**
+
+As a last resort, you may choose to override variables with inline styles. This could be desirable if you only need one component to deviate from the design system. Note that this incurs some level of risk, especially related to colors, as you are opting out of the color combinations in the system.
+
+
+```
+  <pfe-cta style="--pfe-cta--BackgroundColor:pink">
+```
+
+
+
+## FAQ
+
+
+### **Should I use on=dark or color=darkest on my container?  \
```

Comment:
Could we use pfe-accordion for displaying this?

Reply:
not currently; these pages arent loading the web component JS files